File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/E. Max/main.dart
import 'dart:io';
import 'dart:math';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  int mx = 0;
  List<String> input = stdin.readLineSync()!.split(" ");

  while (n > 0) {
    int x = int.parse(input[--n]);
    mx = max(mx, x);
  }
  print(mx);
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/E. Max/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/E

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemE
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            uint max=0;
            string[] input = Console.ReadLine().Split(" ");
            while(n > 0){
                uint x = uint.Parse(input[--n]);
                max = Math.Max(max, x);
            }
            Console.WriteLine(max);
        }
    }
}
